testhelper/destination/rudderserver.go

```diff
@@ -0,0 +1,136 @@
+package main_test
+
+import (
+	"database/sql"
+	_ "encoding/json"
+	"fmt"
+	_ "github.com/Shopify/sarama"
+	_ "github.com/lib/pq"
+	"github.com/minio/minio-go"
+	"github.com/ory/dockertest"
+	dc "github.com/ory/dockertest/docker"
+	"github.com/phayes/freeport"
+	"log"
+	"net/http"
+	"strconv"
+)
+
+var (
+	DB_DSN = "root@tcp(127.0.0.1:3306)/service"
+)
+
+type ServerTest struct {
+	db *sql.DB
+}
+
+type MINIO struct {
+	minioEndpoint   string
+	minioBucketName string
+}
+
+func SetJobsDB(Test *Test) (*sql.DB, *dockertest.Resource) {
```

Comment:
Even if we currently only use sql.DB, is better to return the whole `type ServerTest struct`. Making the `db` exportable `DB`. 

This way if we need to return something extra like:
* database name
* username
* password
* port

We can easily add them.

I would suggest even now to return DB_DSN in form of `url.URL`